auth/src/include/firebase/auth.h

```diff
@@ -631,6 +632,14 @@ class Auth {
   void RemoveIdTokenListener(IdTokenListener* listener);
 #endif  // not SWIG
 
+#if !defined(DOXYGEN)
```

Comment:
Should probably hide from SWIG for now (unless the plan is for this to also be visible from Unity)

Reply:
done